### PR TITLE
Model

### PR DESCRIPTION
**Pull request by Jonas-Aramayo**
hey guys,

I created a new class is called search, so far it only do the list of all files using a path.
Could you please look at this when you have a chance?

### CODE REVIEW

File: src/com/jalasoft/search/Model/Search.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    private String getFilePath() {
+        return folderPath;
+    }
+
+    void doListFiles(){
+         //Read file
+        File file = new File(folderPath);
+
+        //folderExist
+        if (file.exists() == true) {
```
Comment (GabrielRendon) [inline src/com/jalasoft/search/Model/Search.java]: I think this is part of the validator and not the model

File: src/com/jalasoft/search/Model/Search.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            //isDirectory
```
Comment (GabrielRendon) [inline src/com/jalasoft/search/Model/Search.java]: Same here

File: src/com/jalasoft/search/Model/Search.java (hunk 1 above)
Comment (Jonas-Aramayo) [reply src/com/jalasoft/search/Model/Search.java]: Yes, you are right, I will remove it in the next pull request.

File: src/com/jalasoft/search/Model/Search.java (hunk 2 above)
Comment (Jonas-Aramayo) [reply src/com/jalasoft/search/Model/Search.java]: the same action below.

**Pull request closed without merging**